src/Settings/SettingsHelper.php

```diff
@@ -1064,4 +1064,53 @@ function ($option) use ($formId, $key) {
 			],
 		];
 	}
+
+	/**
+	 * Output array - form selection additional.
+	 *
+	 * @param string $formId Form ID.
+	 * @param array<string, mixed> $items Items from cache data.
+	 * @param string $selectedFormId Selected form id.
+	 * @param string $key Settings key used.
+	 *
+	 * @return array<int, array<string, array<int|string, array<string, mixed>>|bool|string>>
+	 */
+	private function getOutputFormSelectionAdditional(
+		string $formId,
+		array $items,
+		string $selectedFormId,
+		string $key
+	): array {
+		return [
+			[
+				'component' => 'select',
+				'selectName' => $this->getSettingsName($key),
+				'selectId' => $this->getSettingsName($key),
+				'selectFieldLabel' => \__('Select sub form', 'eightshift-forms'),
```

Comment:
I am not too sure what is a "sub form", but the name should probably be changed to something a bit more intuitive and appropriate.